Test/Install-WhiskeyDotNetTool.Tests.ps1

```diff
@@ -19,6 +19,8 @@ function Init
     $script:globalDotNetDirectory = Join-Path -Path $TestDrive.FullName -ChildPath 'GlobalDotNetSDK'
     $script:version = $null
     $script:workingDirectory = $null
+    Mock -CommandName 'Get-Command' -ModuleName 'Whiskey' -ParameterFilter { $Name -eq 'dotnet' } -MockWith { $null }.GetNewClosure()
+    Mock -CommandName 'dotnet' -ModuleName 'Whiskey' -MockWith { 'return some strings'; $global:LASTEXITCODE = 1 }.GetNewClosure()

```

Comment:
A cleaner way to mock the exit code is to run a process that actually exits with the desired exit code:

```powershell
Mock -CommandName 'dotnet' -ModuleName 'Whiskey' -MockWith { cmd /c exit 1 }
```